amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+
+import BookmarkIcon from 'components/Bookmark/BookmarkIcon';
+import { getSourceDisplayName } from 'config/config-utils';
+import BadgeList from 'features/BadgeList';
+import { ResourceType } from 'interfaces/Resources';
+import { LineageItem } from 'interfaces/TableMetadata';
+import { RightIcon } from 'components/SVGIcons';
+
+export interface LineageListProps {
+  items: LineageItem[];
+}
+
+const LineageList: React.FC<LineageListProps> = ({
+  items,
+}: LineageListProps) => (
+  <div className="list-group">
+    {items.map((table, index) => {
+      const { cluster, database, schema, name } = table;
+      const link = `/table_detail/${cluster}/${database}/${schema}/${name}?index=${index}&source=table_lineage_list`;
+      // TODO - test badges
+      const badges = table.badges.map((badgeName) => ({
```

Comment:
you don't need this here, as you should pass the complete list of badges to the `BadgeList` component.